install/kubernetes/cilium/README.md

```diff
@@ -445,6 +445,7 @@ contributors across the globe, there is almost always someone available to help.
 | gatewayAPI.secretsNamespace.sync | bool | `true` | Enable secret sync, which will make sure all TLS secrets used by Ingress are synced to secretsNamespace.name. If disabled, TLS secrets must be maintained externally. |
 | gatewayAPI.xffNumTrustedHops | int | `0` | The number of additional GatewayAPI proxy hops from the right side of the HTTP header to trust when determining the origin client's IP address. |
 | gke.enabled | bool | `false` | Enable Google Kubernetes Engine integration |
+| healthCheckICMPProbesCount | int | `1` | Number of ICMP requests sent for each health check before marking a node or endpoint unreachable. |
```

Comment:
Would it make sense to name the option `healthCheckICMPFailureThreshold`?

To make it consistent with Kubernetes probes [failureThreshold](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#:~:text=value%20is%201.-,failureThreshold,-%3A%20After%20a%20probe).

Reply:
Agree, will update :+1: